src/Sulu/Bundle/MediaBundle/Resources/config/image-formats.xml

```diff
@@ -86,4 +77,13 @@
         </meta>
         <scale x="400" y="400" mode="inset"/>
     </format>
+    <format key="sulu-100x100" internal="true">
+        <meta>
+            <title lang="en">Contact card (Sulu)</title>
+            <title lang="de">Personenkarte (Sulu)</title>
+            <title lang="fr">Carte personnelle (Sulu)</title>
+            <title lang="nl">Contact kaart (Sulu)</title>
```

Comment:
Do you need them in size 100x100? And I would rename the titles to avoid misunderstandings (this format is not used for contact cards).

Reply:
I'd say we move that format to the original place (to avoid merge conflicts in the future), and have another look at all the titles and formats name when we handle #2639.